epub34/rs/index.html

```diff
@@ -2162,10 +2159,10 @@ <h2>Processing <code>property</code> values</h2>
 						<summary>Explanation</summary>
 						<p>The path-relative-scheme-less-URL string definition has a number of restrictions that apply
 							to the reference whether it has a prefix or not.</p>
-						<p>For example, the reference must not begin with a [=URL-scheme string=] followed by a colon.
+						<p>For example, the reference cannot begin with a [=URL-scheme string=] followed by a colon.
```

Comment:
```suggestion
						<p>For example, better to avoid a reference beginning with a [=URL-scheme string=] followed by a colon.
```

The word "cannot" does not sound equivalent to the "must not" that it is supposed to replace. "Cannot" sounds like "not able to" to me.

(There may be a better replacement than mine, though.)

Reply:
I think "better to avoid" makes it more permissive than it is. That would be guidance for a "should not". We could say:
```suggestion
						<p>For example, the reference is not allowed to begin with a [=URL-scheme string=] followed by a colon.
```

Reply:
Sounds good to me.